worksheet_B/readme.MD

```diff
@@ -0,0 +1,38 @@
+code:
+```
+
+START
+
+Attempts=0
+i=0
+Likeness=0
+OriginalWord=Generate
+
+while (attempts <= 4):
+	Output ("enter your first attempt: ")
+	UserEntery=UserInput
+	attempts+1
+	
+	For (each character in OriginalWord):
+		i+1
+		if (letter == letter in UserEntery)
```

Comment:
Need to make it clear here that you're comparing the ith letter in OriginalWord to the ith letter in UserEntry